tools/clang/lib/SPIRV/EmitSPIRVAction.cpp

```diff
@@ -230,22 +229,27 @@ class SPIRVEmitter : public ASTConsumer {
     }
   }
 
-  void doStmt(Stmt *stmt) {
-    if (auto *retStmt = dyn_cast<ReturnStmt>(stmt)) {
+  void doStmt(const Stmt *stmt) {
```

Comment:
I guess that makes sense.  Those do<construct> things basically mirror Handle<construct> which have well defined meaning from the superclass.